packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3inbox/core",
-  "version": "1.0.3",
+  "version": "1.1.0",
```

Comment:
Do we align this package with notify-client or just coincidence?

Reply:
Just a coincidence, bumped a `minor` version because we added a feature